src/main/java/org/italiangrid/storm/webdav/config/SAConfigurationParser.java

```diff
@@ -68,6 +75,27 @@ public SAConfigurationParser(ServiceConfiguration sc) {
       }
 
       OwnerStorageAreaInfo saInfo = ConfigFactory.create(OwnerStorageAreaInfo.class, p);
+      if (!statAlreadyChecked && saInfo.tapeEnabled()) {
+        try {
+          Process process = Runtime.getRuntime().exec("stat -c %b " + f.getPath());
```

Comment:
this check shouldn't be here. even if you want to do it in during the SA configuration, you just need to get a file in that directory and do the check, outside of this loop

Reply:
yes, I know, but it's not nice. at least hide the check behind a function. and it doesn't really matter if the sa is a tape one or not.